docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Retrieve and use payment details from third-party payment providers
+description: Learn how to retrieve and use payment details from a third-party payment service providers
+draft: true
```

Comment:
Does this need to be published or just merged?

Reply:
Should be published as well